pkg/kp/pcstore/consul_store_test.go

```diff
@@ -255,18 +256,17 @@ func TestWatch(t *testing.T) {
 
 	select {
 	case watchedPC := <-watch:
-		watched = &watchedPC
+		watched = watchedPC
 	case <-time.After(5 * time.Second):
-		quit <- struct{}{}
 		t.Fatal("nothing on the channel")
 	}
 
-	if watched == nil {
-		t.Fatalf("Expected to get a watched PodCluster, but did not")
+	if len(watched.Clusters) != 1 {
```

Comment:
This should have been a comment from when this file was first added, but I guess I gotta do it retroactively: Check that `watched.Err` is nil?